lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -5275,6 +5375,7 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
     if (origResult->getType().isObject()) {
       // If original result is a `tuple_extract`, materialize adjoint value of
       // `ai` and extract the corresponding element adjoint value.
+      // TODO: Handle DTI
```

Comment:
Unaddressed todo comment - remove it or address it?

Reply:
Good catch. This is actually dead code. I removed the special `tuple_extract` handling completely.